tests/mod.rs

```diff
@@ -94,6 +94,25 @@ where
     // `FailurePersistence`s for each test
 }
 
+/// Validates that replay-from-seed works by running a failing seed found by a random scheduler for one
```

Comment:
Nit: `replay-from-seed` isn't anything. Either name the function (`replay_random_from_seed`) or "abstract over it"; "the replay from seed functionality".